packages/core/src/components/cms/global/Components.ts

```diff
@@ -30,13 +31,13 @@ const Footer = dynamic(
 )
 
 const COMPONENTS: Record<string, ComponentType<any>> = {
-  Alert,
-  Navbar,
-  RegionBar,
-  RegionPopover,
-  CartSidebar, // out of viewport
-  RegionModal, // out of viewport
-  Footer, // out of viewport
+  [getComponentKey(Alert, 'Alert')]: Alert,
```

Comment:
question: would that be a breaking change? The store would have to update their CUSTOM_COMPONENTS to match this pattern?